### ISSUE TITLE
Win Compile Error (BB #449)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#449]
This issue was migrated from bitbucket.
**Original Reporter:** ScullyNY
**Original Date:** 12.04.2013 11:49:50 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/449

<hr>

The Cange from Zaffy in src/oregoncore/CliRunnable.cpp
https://bitbucket.org/oregon/oregoncore/commits/62520d878528641f9c2ea4e12890ac8c93cf074e

```
#!c++

-    char commnadbuf[256];

+    char commnandbuf[256];
```

make some Compile Errors. (typing error??)

Test it with: 

```
#!c++

-    char commnandbuf[256];

+    char commandbuf[256];
```

It's work (no error).

**I'm new, hope its right! Please give me a feedback!**

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** ScullyNY
**Original Date:** 12.04.2013 11:50:22 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/449/#comment-3722476

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** ScullyNY
**Original Date:** 12.04.2013 11:51:32 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/449/#comment-3722491

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 12.04.2013 12:08:03 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/449/#comment-3722655

<hr>

lol ! Damn it :)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** ScullyNY
**Original Date:** 12.04.2013 18:13:26 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/449/#comment-3729096

<hr>

It's right? ;)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 17.01.2014 08:32:43 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/449/#comment-7783775

<hr>

Resolved in 748ede2
